### PR TITLE
Placate codacy

### PR DESCRIPTION
**Pull request by aflag**
Closes: https://github.com/ScottishCovidResponse/SCRCIssueTracking/issues/593

I have fixed the trivial ones and told pylint to ignore the things I thought were not valid complaints. I have created this ticket for something that'd take longer to fix (and I'm not entirely sure if we care): https://github.com/ScottishCovidResponse/SCRCIssueTracking/issues/633

The code duplication in actual code is 0. Codacy complains about code duplication in tests, but I consider code duplication in tests good practice.

### CODE REVIEW

File: tests/unit/test_loaders.py [hunk 1]
```diff
@@ -332,7 +332,7 @@ def test_AgeRange_different_ranges(range_a, range_b):
 
     assert hash(a) != hash(b)
     assert a != b
-    assert not (a == b)
+    assert not a == b  # pylint: disable=unneeded-not
```
Comment (bobturneruk) [inline tests/unit/test_loaders.py:335]: Why do we need both these lines? Maybe I'm being daft but they superficially look like they're doing the same thing. Is it to do with how `=` and `!=` are defined for an `AgeRange`?

**Comment by bobturneruk** [comment]
Formatting of examples in [simple_network_sim.loaders.MixingMatrix](https://github.com/ScottishCovidResponse/simple_network_sim/blob/63f4406e4da566e4977996ce934497de480d8e77/simple_network_sim/loaders.py#L364) could perhaps be improved by making it more like [common.lazy](https://github.com/ScottishCovidResponse/simple_network_sim/blob/63f4406e4da566e4977996ce934497de480d8e77/simple_network_sim/common.py#L37).

File: tests/unit/test_loaders.py (hunk 1 above)
Comment (aflag) [reply tests/unit/test_loaders.py:335]: Maybe we could call `a.__eq__(b)` to be more specific. `!=` and `==` call different methods.

**Comment by aflag** [comment]
Regading the bad-continuation. I think I have a better fix for that. Maybe just using 8 spaces instead should fix it.

File: tests/unit/test_loaders.py (hunk 1 above)
Comment (bobturneruk) [reply tests/unit/test_loaders.py:335]: I think leave as is. I was just curious.

**Comment by FiodorG** [comment]
Sorry about the conflicts that's me. The bad continuation is just 8 spaces for next line function parameters yes. To be honest I was skeptical but it does look neater with 8 spaces.

**Pull request merged**